reader-gtfs/src/main/java/com/graphhopper/reader/gtfs/CustomGraphHopperGtfs.java

```diff
@@ -0,0 +1,230 @@
+/*
```

Comment:
This is just pulled from gh, yeah? did you need to modify anything?

Reply:
Essentially, yes, with minor updates. Added javadoc to this class to specify what was changed/what wasn't